invenio_sipstore/api.py

```diff
@@ -225,20 +229,22 @@ def create(cls, pid, record, archivable, create_sip_files=True,
         of the files in the record, along with ``SIPMetadata`` for the
         metadata.
         Those objects are not returned by this function but can be fetched by
-        the corresponding RecordSIP attributes 'sip', 'sip.files' and
-        'sip.metadata'.
+        the corresponding RecordSIP attributes ``sip``, ``sip.files`` and
+        ``sip.metadata``.
+
         :param pid: PID of the published record ('recid').
-        :type pid: `invenio_pidstore.models.PersistentIdentifier`
+        :type pid: :py:class:`invenio_pidstore.models.PersistentIdentifier`
         :param record: Record for which the SIP should be created.
-        :type record: `invenio_records.api.Record`
+        :type record: :py:class:`invenio_records.api.Record`
         :param bool archivable: tells if the record should be archived.
-        Usefull when ``Invenio-Archivematica`` is installed.
+            Usefull when ``Invenio-Archivematica`` is installed.
         :param bool create_sip_files: If True the SIPFiles will be created.
         :returns: RecordSIP object.
         :rtype: :py:class:`invenio_sipstore.api.RecordSIP`
         """
         files = record.files if create_sip_files else None
-        metadata = {'json': json.dumps(record.dumps())}
+        mtype = SIPMetadataType.get_from_schema(record['$schema'])
+        metadata = {mtype.name: json.dumps(record.dumps())}
```

Comment:
TODO: use the future "filename" field from the metadatatype table instead of mtype.name